apps/learninglocker/templates/app/dc_worker.yml.j2

```diff
@@ -0,0 +1,5 @@
+{% set service_variant = "worker" %}
+{% set learninglocker_command = "worker/dist/server" %}
+{% set learninglocker_port = "8080" %}
```

Comment:
This shoudn't be an integer and configurable?

Reply:
it should not have listening port. It a copy past fail.

Reply:
IC